### PR TITLE
Ops 111 bootstrap

### PR DESCRIPTION
**Pull request by yellowmegaman**
**- What I did**
Added option to generate sane names for certificates
**- How I did it**
Added `--propername` flag for bootstrap subcommand
**- How to verify it**
run insolar bootstrap -p true

By default propername = false, so it won't affect existing setups.

### CODE REVIEW

**Comment by yellowmegaman** [comment]
image

File: applicationbase/bootstrap/keys.go [hunk 1]
```diff
@@ -89,8 +90,14 @@ func createKeysInDir(
 			return nil, errors.Wrap(err, "[ createKeysInDir ] couldn't marshal keys")
 		}
 
-		inslogger.FromContext(ctx).Info("Genesis write key " + filepath.Join(dir, keyname))
-		err = makeFileWithDir(dir, keyname, result)
+		if properName {
```
Comment (SergeyRadist) [inline applicationbase/bootstrap/keys.go]: You can di like this
if properName {
    keyname = n.CertName
}
and then log and makeFileWithDir once

Comment (yellowmegaman) [reply applicationbase/bootstrap/keys.go]: Nice! Thanks

Comment (yellowmegaman) [reply applicationbase/bootstrap/keys.go]: Done

**Pull request merged**